### ISSUE TITLE
423399 - balance still showing in all orders

### ISSUE THREAD

**Issue by ali206** [206Tours/DB#1094]
c22c8ad2580e7ee25d8f583874195d1c

![image](https://cloud.githubusercontent.com/assets/8440903/14710669/4ad6be82-07a4-11e6-9c71-e25d636e7ca8.png)

![image](https://cloud.githubusercontent.com/assets/8440903/14710674/516cacb6-07a4-11e6-80c1-f6439152f440.png)

**Comment by bradatkins** [comment]
Closing, hasn't happened since

**Comment by ali206** [comment]
Yes it has...?? Commissions are still not reflecting correctly in All Orders

**Comment by bradatkins** [comment]
Examples please. You'll also likely need to provide the logic because I swear we did this already.

Specifically examples where the balance owing isn't matching the invoice

**Comment by ali206** [comment]
We've been doing this but it's hasn't been 100% correct.  

Currently it seems to be an issue with NET and people not set up with a comm structure

Jimmy is NET so we apply commission to his invoice, invoice is correct showing PIF but in all orders balance due is commission that's been applied:

![image](https://cloud.githubusercontent.com/assets/8440903/20350391/bc7b6e2a-abdc-11e6-8bda-4491f06dfe15.png)


![image](https://cloud.githubusercontent.com/assets/8440903/20350368/ab1d330c-abdc-11e6-9e7d-bd763af98b5a.png)


Another ex - don't think he's set up gross or net
![image](https://cloud.githubusercontent.com/assets/8440903/20350545/5407743c-abdd-11e6-82b3-b167ac4aeaa0.png)

![image](https://cloud.githubusercontent.com/assets/8440903/20350552/5b0deaa4-abdd-11e6-9ce3-497a0e7e9885.png)


![image](https://cloud.githubusercontent.com/assets/8440903/20350755/14b28370-abde-11e6-8a54-1a6957288c70.png)


![image](https://cloud.githubusercontent.com/assets/8440903/20350759/1a51e794-abde-11e6-9dcf-f8fe019cfe2b.png)

**Comment by Irena206** [comment]
Heres a gross example:

![image](https://cloud.githubusercontent.com/assets/14838466/20359201/d22f6c06-abfc-11e6-995d-00b0c1f5667e.png)


![image](https://cloud.githubusercontent.com/assets/14838466/20359216/e8fb349c-abfc-11e6-9039-14b30b9ba23d.png)

**Comment by bradatkins** [comment]
No agent attached: 

![image](https://cloud.githubusercontent.com/assets/8440255/20653938/6fb0ecfa-b4e2-11e6-84f1-00a676f7e26f.png)

**Comment by ali206** [comment]
So for 423399 - The group leader or priest applied his commission to one of the passengers so there will never be an agent to select. But this still needs to show on the invoice as a "payment" made.


426085 - agent is selected and it's showing as a negative #:

![image](https://cloud.githubusercontent.com/assets/8440903/20796749/9b7fe1a4-b7a5-11e6-81ca-52d051ca9f13.png)

![image](https://cloud.githubusercontent.com/assets/8440903/20796764/a92f28e6-b7a5-11e6-95e9-6afab9d639f3.png)
